plugins/dell/fu-plugin-dell.c

```diff
@@ -804,6 +805,37 @@ fu_plugin_update_offline (FuPlugin *plugin,
 	return TRUE;
 }
 
+void
+fu_plugin_device_registered (FuPlugin *plugin, FuDevice *device)
+{
+	FwupdDeviceFlags flags = 0;
+	const gchar *metadata_output;
+	g_autofree gchar *vendor_id = NULL;
+	g_autofree gchar *device_id = NULL;
+
+	flags = fu_device_get_flags (device);
+
+	/* thunderbolt plugin */
+	if (g_strcmp0 (fu_device_get_plugin (device), "thunderbolt") &&
```

Comment:
You're right i'm missing a == 0 or ! (I need to get a try on real HW this afternoon).